ts/features/fci/components/ErrorComponent.tsx

```diff
@@ -1,12 +1,17 @@
 import * as React from "react";
-import { SafeAreaView } from "react-native";
+import { SafeAreaView, View } from "react-native";
```

Comment:
Should we use the `SafeAreaView` exposed by `react-native-safe-area-context`? 
The `SafeAreaView` from `react-native` only works in iOS.
This is just an example but we are using the wrong one in many other places which can be found by searching for `SafeAreaView` and filtering only the folder `ts/features/fci`.

Reply:
The `react-native-safe-area-context` is not yet totally used in the io-app codebase. Replacing this component in place of safeareaview causes additional padding to be added to the top. it could be solved by using the edges prop.
`<SafeAreaView edges={['top', 'left', 'right']}></SafeAreaView>`
The question is: looking to our DS is the right way to use SafeAreaView?

cc @LazyAfternoons @dmnplb

Reply:
@hevelius @LazyAfternoons In some components developed for the DS we use native RN Views (e.g: `ScrollView`) with a bottom padding value set by the `insets` provided by the `useSafeAreaInsets()` hook (from the same package). So, to add the bottom padding to a `ScrollView`:
```jsx
    <ScrollView
      contentContainerStyle={{
        paddingBottom: insets.bottom
      }}
    >
    […]
```